content/english/sql_workshop/activity1/_activity1.md

```diff
@@ -22,9 +22,10 @@ commands to help save the Planet of Fun!
 You are given information about the Galaxy and want to look for the Planet of Fun. To do this,
  the Galactic Federation has given you your first SQL command: SELECT. 
 
-**What the command does:** The SELECT command allows you to choose what data to show from a database.
+### What the command does: 
```

Comment:
I realized the names of the files should be improved - can you name these files _index.md?

This is so that the URL of the workshop becomes /sql_workshop/activity1 instead of /sql_workshop/activity1/_activity1

Reply:
I'll make the change!